src/index.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import "babel-polyfill";
```

Comment:
fwiw `async/await` is supported through some Babel directibe added to `.babelrc` directly, no need for dedicated imports. At least that's what worked in kinto-http.js.